docs/user-guide/configure.md

```diff
@@ -299,6 +299,7 @@ The following rules are configured via the `languageOptions` property:
 
 - [`at-rule-descriptor-no-unknown`](../../lib/rules/at-rule-descriptor-no-unknown/README.md)
 - [`at-rule-descriptor-value-no-unknown`](../../lib/rules/at-rule-descriptor-value-no-unknown/README.md)
+- [`at-rule-no-unknown`](../../lib/rules/at-rule-no-unknown/README.md)
 - [`at-rule-prelude-no-invalid`](../../lib/rules/at-rule-prelude-no-invalid/README.md)
 - [`declaration-property-value-no-unknown`](../../lib/rules/declaration-property-value-no-unknown/README.md)
```

Comment:
[note] Personally, I don't feel this rule list is no longer necessary because each rule document has a link here, and it seems to become hard to maintain the list. Not blocker.

Reply:
Thanks for creating an issue. Keeping the list for a bit longer seems fine to me 👍🏼